Sources/Build/BuildPlan.swift

```diff
@@ -674,9 +675,15 @@ public final class SwiftTargetBuildDescription {
         let path = derivedSources.root.appending(subpath)
         try fs.writeIfChanged(path: path, bytes: stream.bytes)
     }
-
+    
+    public static func checkSupportedFrontendFlags(flags: Set<String>, fs: FileSystem) throws -> Bool {
```

Comment:
I wonder if a thrown error here should maybe be treated the same as not supporting the flags.